_sass/tools/_mixins.scss

```diff
@@ -0,0 +1,44 @@
+// gradient mixins
+@mixin c-hero-background {
+  width: 100%;
+  height: 100%;
+  background: url('../assets/logo-background.svg'), $top-gradient, url('../assets/homepage-clouds.jpg');
+  background-repeat: no-repeat;
+  background-size: 250%, cover, 200%;
+  background-position: center -10%, center center ,bottom center;
+  padding: 6rem 0;
+}
+
+// header font-styles
+@mixin main-heading-font {
+  font-family: $heading-font;
+  font-weight: 800;
+  font-style: normal;
+  font-size: 2rem;
+  line-height: 1.3;
+  margin: 0;
+  max-width: 299px;
```

Comment:
what is this max-width for? 🙂

Reply:
this is a set max-width for the main heading to stay as 3 words in one line and then 2 in last line. so I don't have just one word in the last line, if that makes sense

Reply:
okay so, I moved it inside the rule so it overwrites that rule. since this mixin will be used for other pages that don't necessarily need that max-width for the heading